lte/gateway/docker/s1ap/start-s1ap.sh

```diff
@@ -10,6 +10,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Create docker networks
+docker network inspect s1&>/dev/null || {
+docker network create --driver=bridge --subnet=192.168.60.0/24 --gateway=192.168.60.142 sgi && \
+docker network create --driver=bridge --subnet=192.168.128.0/23 --gateway=192.168.129.1 s1 && \
+docker inspect s1aptester&>/dev/null && docker rm s1aptester && \
```

Comment:
Above there was the strategy of one command per line, which is much more readable. Imho we should stick to that convention here.